bench/zane_bench.c

```
@@ -183,78 +183,174 @@ static void bench_pool_shutdown(void) {
+    return ((uint32_t)(bo >> 20) << ZM_WORDBITS) | (uint32_t)((bo & (ZM_CHUNK - 1)) >> 3);
+}
+static inline void *zm_resolve(uint32_t seg) {
+    uint32_t chunk_id = seg >> ZM_WORDBITS;
+    assert(chunk_id < ZM_NCHUNKS);
+    return zm.dir[chunk_id] + ((size_t)(seg & ZM_OFFMASK) << 3);
+}
+
+static void zm_free_cell(ZRef ref);
+
+static void zm_reset(void) {
+    for (size_t i = 0; i < zm.nretire; i++) zm_free_cell(zm.retire[i]);
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)